src/client/components/quick-filter/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './index.scss';
+import {Checkbox, Card, Image, Button, CardContent, Grid, Container, Icon, Header, Label, Content } from 'semantic-ui-react';
+
+class Quickfilter extends React.Component {
+
+    constructor(props){
+        super(props);
+
+        const a = function(){
```

Comment:
You should name use such names for your variables. Each name mush have a meaning. If it's a function, better use a verb for it's name. If it's a boolean properly - adjective. If it's an object or class - noun.